logstash-core/lib/logstash/environment.rb

```diff
@@ -34,6 +34,7 @@ module Environment
   end
 
   [
+            Setting::String.new("on_superuser", "WARN", true, ["WARN", "BLOCK", "ALLOW"]),
```

Comment:
Even if we open the door to extension on this setting by having it be a String, can we still just keep "BLOCK" and "ALLOW" (ALLOW+WARN) and drop warn?
I'm convinced the endgame here is to drop support for running as root in the future, so "allowing without warning" doesn't make sense IMO.

Reply:
Thanks @jsvd ,
Since ALLOW & WARN are relatively close, merging them seems reasonable to me as well.